### ISSUE TITLE
Issues running/installing under Raspbian Bullseye

### ISSUE THREAD

**Issue by kodlz** [CoolerMasterTechnology/Pi-Tool#21]
### Discussed in https://github.com/CoolerMasterTechnology/Pi-Tool/discussions/20

<div type='discussions-op-text'>

<sup>Originally posted by **GaelNguyen** November 13, 2021</sup>
I just loaded the latest version of Raspbian and the Pi Tool seems to install without a problem however when I try to open the app it won't run. Has anyone else found issues with the latest versions of Raspbian?</div>

**Comment by kodlz** [comment]
HI,
I have similar issue,
I just try to install it on new Raspberry OS 11 **arm64**.
It looks like one of lib is not available for Bullseye

    `pi-tool : Depends: libappindicator1 but it is not installable`

probably that is reason why is not possible run Pi-tool.

log from installation is attach
[pi-tool_install.log](https://github.com/CoolerMasterTechnology/Pi-Tool/files/7621329/pi-tool_install.log)
ed

**Comment by kodlz** [comment]
Hi Team,
Any update about issue?

**Comment by stephanepapin** [comment]
> HI, I have similar issue, I just try to install it on new Raspberry OS 11 **arm64**. It looks like one of lib is not available for Bullseye
> 
> ```
> `pi-tool : Depends: libappindicator1 but it is not installable`
> ```
> 
> probably that is reason why is not possible run Pi-tool.
> 
> log from installation is attach [pi-tool_install.log](https://github.com/CoolerMasterTechnology/Pi-Tool/files/7621329/pi-tool_install.log) ed

Same issue :( I've bougth Case for my RPI-4-8gb with SSD boot

**Comment by wck86** [comment]
I'm having the same issue and am looking for a fix. Will advise if I find it.

**Comment by wck86** [comment]
Fix for users of the current 64 bit Raspberry Pi OS: 
In terminal, from the home directory (user@hostname:~ $), enter the following commands below one after the other, confirming as needed, 

(Install dependencies)
sudo apt install libdbusmenu-glib4 libdbusmenu-gtk4 libfontconfig1 -y

(Download and install dependency not in current repos)
wget http.us.debian.org/debian/pool/main/libi/libindicator/libindicator7_0.5.0-4_arm64.deb
sudo dpkg -i libindicator7_0.5.0-4_arm64.deb

(Download and install dependency not in current repos)
wget http.us.debian.org/debian/pool/main/liba/libappindicator/libappindicator1_0.4.92-7_arm64.deb
sudo dpkg -i libappindicator1_0.4.92-7_arm64.deb

(Re-run Pi Tool installation script)
./pi-tool-install.sh

**Comment by wck86** [comment]
![image](https://user-images.githubusercontent.com/6379471/150654655-f34eae6b-8486-48e4-b137-94679f54951b.png)

**Comment by kodlz** [comment]
Hi @wck86 
Thank you for you solution, but unfortunately is not working for me. I'm not able install libappindicator1.

` $ sudo dpkg -i libappindicator1_0.4.92-7_arm64.deb
Selecting previously unselected package libappindicator1:arm64.
(Reading database ... 58457 files and directories currently installed.)
Preparing to unpack libappindicator1_0.4.92-7_arm64.deb ...
Unpacking libappindicator1:arm64 (0.4.92-7) ...
dpkg: dependency problems prevent configuration of libappindicator1:arm64:
 libappindicator1:arm64 depends on libgtk2.0-0 (>= 2.24.32); however:
  Package libgtk2.0-0 is not installed.
 libappindicator1:arm64 depends on libindicator7 (>= 0.4.90); however:
  Package libindicator7 is not installed.

dpkg: error processing package libappindicator1:arm64 (--install):
 dependency problems - leaving unconfigured
Processing triggers for libc-bin (2.31-13+rpt2+rpi1+deb11u2) ...
Errors were encountered while processing:
 libappindicator1:arm64
`

**Comment by kodlz** [comment]
Hi @wck86
Thank you for your solution, 
it's working now ;)

**Comment by stephanepapin** [comment]
I will try today. But I've a question does it possible to set parameters for button and overclocking in command line? Because my RPI4-8G don't have the desktop installed.

**Comment by kodlz** [comment]
@stephanepapin you don't need desktop for managing pi-tool. Only what is required is use terminal with x11. I'm using MobaXterm.

**Comment by stephanepapin** [comment]
> 
I don't have GUI installed on my RPI. I'm using SSH wih Putty

**Comment by kodlz** [comment]
yes, that is what I talk about.  You don't need desktop/GUI installed on RPI. Just use  X11 connection to RPI.
in your case: https://superuser.com/questions/119792/how-to-use-x11-forwarding-with-putty

**Comment by stephanepapin** [comment]
> yes, that is what I talk about. You don't need desktop/GUI installed on RPI. Just use X11 connection to RPI. in your case: https://superuser.com/questions/119792/how-to-use-x11-forwarding-with-putty
Sure.... but without Graphic GUI engine on RPI I'm in doubt into how Pi Tool will be displayed
